### ISSUE TITLE
No visual indicator when image-based links are in tab focus

### ISSUE THREAD

**Issue by chyatt** [ccswbs/hjckrrh#536]
Reproducible in Chrome at least.

E.g. engineering, United Way Idol pages

**Comment by brharp** [comment]
Disabling this rule:

a:focus {
    outline: thin dotted;
}

Seems to fix the problem in Chrome. Do we understand the problem here? Is “thin dotted” a valid value for the “outline” property? What happens if we remove the rule for “a:focus” altogether? Does anything else break?

Brent

**Comment by tqureshi-uog** [comment]
It seems this rule was added in order to force Chrome to use a dotted-line focus indicator (as opposed to its default of a thick blue border). Disabling it has no adverse effect whatsoever - FF continues to use its own default dotted line focus indicator on all links (text and images), while Chrome defaults to the blue border.

With this rule in place, Chrome outlines text links with a thin dotted line, but image links get no focus indicator. Perhaps an extra CSS rule could be added that explicitly assigns a thin dotted focus indicator to image links, but it looks like the original rule was for minor aesthetic reasons only and it's better to just let the browser defaults for focus outlines stay in place.

**Comment by brharp** [comment]
@tqureshi-uog Then please create a pull request with the rule disabled, so we can review / test the change.

**Comment by brharp** [comment]
Fixed by pull request #545.

**Comment by mmafe** [comment]
@brharp Hey Brent, has this been released yet? I'm still seeing the issue on client sites. Eg. https://aoda-client.uoguelph.ca/police

**Comment by brharp** [comment]
Merged, but not yet released.


________________________________________
